### ISSUE TITLE
Speaker Support?

### ISSUE THREAD

**Issue by shubshub** [rdepena/node-dualshock-controller#63]
Any chance of adding Speaker support to this library please? Would be very awesome! :D

**Comment by rdepena** [comment]
YES!!, I am chipping a away at further support (Recently added Touchpad support #61), I will use this ticket to document progress/research.

Usually I fall into the following pattern:
- [ ] Investigate if someone has reversed engineered this particular piece of functionality, a good source is  [PSDevWiki](http://www.psdevwiki.com/ps4/Talk:DualShock_4)
- [ ] Start discovery on the HID level: Read/Write/Observe, annotate relevant bytes for feature reports etc.  
- [ ] POC Functionality on Native HID
- [ ] Move to node-hid and this code base.

**Comment by rdepena** [comment]
YES!!, I am chipping a away at further support (Recently added Touchpad support #61), I will use this ticket to document progress/research.

Usually I fall into the following pattern:
- [ ] Investigate if someone has reversed engineered this particular piece of functionality, a good source is  [PSDevWiki](http://www.psdevwiki.com/ps4/Talk:DualShock_4)
- [ ] Start discovery on the HID level: Read/Write/Observe, annotate relevant bytes for feature reports etc.  
- [ ] POC Functionality on Native HID
- [ ] Move to node-hid and this code base.

**Comment by shubshub** [comment]
Will be keen to follow research until its added c:

**Comment by matthewbauer** [comment]
Someone has gotten it to work with ds4drv:

https://github.com/chrippa/ds4drv/pull/80/

**Comment by rdepena** [comment]
Damn that's pretty cool, I will need to check it out.
